pkg/rebuild/debian/strategy.go

```diff
@@ -98,6 +101,16 @@ func (b *Debrebuild) ToWorkflow() *rebuild.WorkflowStrategy {
 		},
 		OutputDir: "out",
 	}
+	// TODO: Maybe there should be a debrebuild minimal that aggressively narrows the TCB of a rebuild
```

Comment:
nit: you mean maybe there should be a separate strategy or a separate param or a separate command?